crates/utils/re_depth_compression/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+//! Depth compression utilities.
+
+pub mod ros_rvl;
```

Comment:
the namespacing here seems rather unnecessary since `ros_rvl` is repeated everywhere. Should flatten it out (also I dislike `pub mod` in general because you never quite know what you're exporting ;))